modules/internal/ChapelLocale.chpl

```diff
@@ -90,20 +90,142 @@ module ChapelLocale {
     return (subloc != c_sublocid_none
             && subloc != c_sublocid_any
             && subloc != c_sublocid_all);
+  
+  /*
+    regular: Has a concrete BaseLocale instance
```

Comment:
Just for clarity, order of values in the comment should match order in the enum declaration.